curvefs/src/tools/curvefs_tool_define.h

```diff
@@ -133,6 +133,14 @@ const char kHostFollowerValue[] = "follower";
 const char kEtcdLeaderValue[] = "StateLeader";
 const char kEtcdFollowerValue[] = "StateFollower";
 
+/* fs type */
+const char kFsTypeS3[] = "s3";
+const char kFsTypeVolume[] = "volume";
+
+/* json type */
+const char kJsonTypeBuild[] = "build";
```

Comment:
What's the difference of json type of 'build' and 'tree'?

Reply:
> What's the difference of json type of 'build' and 'tree'?

build jsonfile can be used to build topology,
tree json file will form a tree of topology according to the structure of cluser, pool, zone, server, metaserver